#include missing hraders
#By B. Hajgato August 31st 2015
-- lib/cpp/antlr/CharScanner.hpp.org   2006-11-01 22:37:17.000000000 +0100
+++ lib/cpp/antlr/CharScanner.hpp       2015-08-31 14:16:49.676709500 +0200
@@ -11,6 +11,8 @@
 #include <antlr/config.hpp>
 
 #include <map>
+#include <fstream>
+#include <strings.h>
 
 #ifdef HAS_NOT_CCTYPE_H
 #include <ctype.h>
--- lib/cpp/src/NoViableAltForCharException.cpp.org 2006-11-01 22:37:17.000000000 +0100
+++ lib/cpp/src/NoViableAltForCharException.cpp     2015-08-31 11:17:31.301226926 +0200
@@ -5,6 +5,9 @@
  * $Id: //depot/code/org.antlr/release/antlr-2.7.7/lib/cpp/src/NoViableAltForCharException.cpp#2 $
  */
 
+#include <fstream>
+#include <strings.h>
+
 #include "antlr/NoViableAltForCharException.hpp"
 #include "antlr/String.hpp"
